EDDiscovery/HistoryList.cs

```diff
@@ -70,6 +71,11 @@ public bool ISEDDNMessage
         public int TravelledMissingjump { get { return travelled_missingjump; } }
         int travelled_jumps;
 
+        private static double last_fuelscooptotal = 0.0;
```

Comment:
I had a quick look at this again this morning and I can change the code to avoid the static, I'll have go tonight.